misc/windows-deploy/amazon-ecs-agent.ps1

```diff
@@ -78,18 +78,7 @@ try {
 
     if([System.Environment]::GetEnvironmentVariable("ECS_ENABLE_TASK_IAM_ROLE", "Machine") -eq "true") {
         LogMsg "IAM roles environment variable is set."
-
-        .\loopback.ps1
         .\hostsetup.ps1
```

Comment:
Does the IAM role test failing issue you mentioned to me with 17.03.1-ee-3, get resolved now ?